frontend/test_runner.go

```diff
@@ -42,11 +44,11 @@ func RunTests(ctx context.Context, client gwclient.Client, spec *dalec.Spec, ref
 		return nil
 	}
 
+	// Force evaluation here so that any errors for the build itself can surface
```

Comment:
Can't see it here, but since you're in the file can you please update the comment to `RunTests` instead of `Run tests`: https://github.com/Azure/dalec/blob/8a9d65bde9a122ed5c9c04190b7e980c3037b092/frontend/test_runner.go#L23